.gitmodules

```diff
@@ -1,6 +1,4 @@
 [submodule "src/jellyfin-web"]
 	path = src/jellyfin-web
 	url = https://github.com/jellyfin/jellyfin-web.git
-[submodule "src/cordova-plugin-chromecast"]
-	path = src/cordova-plugin-chromecast
-	url = https://github.com/jellyfin/cordova-plugin-chromecast.git
+	branch = master
```

Comment:
Files should end with a newline.

Reply:
Hi, it's fixed, please check it, thanks.